src/Components/NavBar/NavBar.js

```diff
@@ -1,18 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { NavLink } from 'react-router-dom';
 import { slide as Menu } from 'react-burger-menu';
+import DropdownWidthMediaQuery from './mediaQueries';
 
 function NavBar({ nameToggle, user }) {
 
   const [displayName, setDisplayName] = useState(null);
+  const [width, setWidth]   = useState(window.innerWidth);
+  const [height, setHeight] = useState(window.innerHeight);
 
   useEffect(() => {
     if (nameToggle) {
       setDisplayName(nameToggle);
     }
+    window.addEventListener("resize", updateDimensions);
+    return () => window.removeEventListener("resize", updateDimensions);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const updateDimensions = () => {
```

Comment:
I haven't used dynamic jsx dimensions like this before, so very cool to see how it's done. This is slick, nice work!